src/codeu/chat/client/core/Context.java

```diff
@@ -47,4 +49,8 @@ public UserContext create(String name) {
     }
     return users;
   }
+
+  public ServerInfo getServerUptime() {
+    return ((View)(view)).getServerUptime();
```

Comment:
Why is this casting necessary?

Casting is usually a bad sign, as it means that you're telling the computer two conflicting things about something's type. In this case, you're telling it that `view` is a `BasicView` (which means it has no `getServerUptime()` method) and you're also telling it that `view` is a `View` (which means it does have `getServerUptime()`).

Can you fix this by adding `getServerUptime()` to the `BasicView` interface? If not, can you fix this by changing the declared type of `view` here?